packages/runtime/container-runtime/src/test/containerRuntime.spec.ts

```diff
@@ -1784,8 +1784,11 @@ describe("Runtime", () => {
 
 				(containerRuntime as any).pendingStateManager = mockPendingStateManager;
 
-				const state = containerRuntime.getPendingLocalState();
-				assert.strictEqual(state, undefined);
+				const state =
+					containerRuntime.getPendingLocalState() as Partial<IPendingRuntimeState>;
+				assert.strictEqual(state.pending, undefined);
```

Comment:
Why do we have this test, given state.pending type disallows undefined? I.e. what exactly it tests?

Reply:
It tests that even in cases where we don't have pending state as ops, we still going to get the sessionExpiryTimerStarted

Reply:
but these asserts don't do anything in fact. I'll just remove them